components/advocates/AdvocateCard.vue

```diff
@@ -1,79 +1,66 @@
 <template>
-  <article class="advocate-card">
-    <div
-      class="advocate-card__picture"
-      :style="`background-image: ${decorate(image)};`"
-    />
-    <div class="advocate-card__copy">
-      <h3 class="advocate-card__name">
-        {{ name }}
-      </h3>
+  <AppCard
+    class="advocate-card"
+    :image="image"
+    :title="name"
+    :tags="formattedRegion"
+  >
+    <div>
       <p class="advocate-card__location">
+        <Map20 class="advocate-card__icon" />
         {{ location }}
       </p>
-      <h3 class="advocate-card__focus-areas-title">
-        Focus areas
-      </h3>
-      <p class="advocate-card__focus-areas">
-        {{ areas }}
+      <p class="advocate-card__contact">
```

Comment:
For some advocates, we don't have the slack contact, could you add a `v-if="slackId"` to only show this block when we have the slack id?